### ISSUE TITLE
How to get a correct Firebase configuration object as JSON?

### ISSUE THREAD

**Issue by filipedelimabrito** [firebase/firebase-tools#2050]
### [REQUIRED] Environment info

**firebase-tools:** 7.15.1

**Platform:** macOS

### [REQUIRED] Test case

From the Firebase CLI (reference [here](https://firebase.google.com/docs/cli#management-commands)) we can obtain a Google services configuration of a Firebase App using the `firebase apps:sdkconfig` command. 

Before this command we could get that configuration using the `firebase setup:web` command but it is now deprecated and its use can brings us some implications (it is missing the `measurementId` which is required to set up the Google Analytics for instance).

Using the previous command with the flag `--json` brings out a correct JSON while the current one doesn't .

### [REQUIRED] Steps to reproduce

Run the `firebase setup:web` with the `--json` flag and you'll have something like that:

```
{
  "status": "success",
  "result": {
    "projectId": "your-project-id",
    "databaseURL": "your-database",
    "storageBucket": "your-storage-bucket",
    "locationId": "a-location-id",
    "apiKey": "your-api-key",
    "authDomain": "your-auth-domain",
    "messagingSenderId": "a-messaging-sender-id"
  }
}

```

Now run the `firebase apps:sdkconfig` with the `--json` flag and you'll have something like that:

```
{
  "status": "success",
  "result": {
    "fileName": "google-config.js",
    "fileContents": "// Copy and paste this into your JavaScript code to initialize the Firebase SDK.\n// You will also need to load the Firebase SDK.\n// See https://firebase.google.com/docs/web/setup for more details.\n\nfirebase.initializeApp({\n  \"projectId\": \"your-project-id\",\n  \"appId\": \"an-app-id\",\n  \"databaseURL\": \"your-database\",\n  \"storageBucket\": \"your-storage-bucket\",\n  \"locationId\": \"a-location-id\",\n  \"apiKey\": \"your-api-key\",\n  \"authDomain\": \"your-auth-domain\",\n  \"messagingSenderId\": \"a-messaging-sender-id\",\n  \"measurementId\": \"a-measurement-id\"\n});\n"
  }
}
```

### [REQUIRED] Expected behavior

`firebase apps:sdkconfig` with the `--json` flag should brings us a correct JSON file like before.

### [REQUIRED] Actual behavior

`firebase apps:sdkconfig` with the `--json` flag brings us a not correct JSON file like before.

**Comment by bkendall** [comment]
I would say that that's a regression. Let me see what I can do real quick...

**Comment by bkendall** [comment]
This has been released in `7.16.0` - check for `sdkConfig` in the json that's returned

**Comment by filipedelimabrito** [comment]
Thanks for working as quick as possible on it.
